OOP_Amazon.py

```diff
@@ -0,0 +1,72 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium import webdriver
+from selenium.webdriver.common.action_chains import ActionChains
+import time
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+class BaseFunction():
+    driver = webdriver.Chrome()
+    driver.maximize_window()
+    TEXT = ""
+
+    def click(self, locator):
+        WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable(locator)).click()
+
+    def input(self, locator, value):
+        WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable(locator)).send_keys(value)
+
+    def get_text(self, locator):
+        self.TEXT == self.driver.find_element_by_css_selector(locator).text
```

Comment:
çift eşit işareti ne için? bi de return text olabilirdi belki. farklı bi yöntem olmuş tabi bilemedim. return self.driver.find_element_by_css_selector(locator).text gibi